docs/guides/setup/installation/components_local/_create_a_virtual_environment.mdx

```diff
@@ -0,0 +1,27 @@
+<!--
+-->
+
+Once you have confirmed that Python 3 is installed locally, you can create a virtual environment with venv.
+
+<details>
+<summary>Python Virtual Environments</summary>
+We have chosen to use venv for virtual environments in this guide, because it is included with Python 3. You are not limited to using venv, and can just as easily install Great Expectations into virtual environments with tools such as virtualenv, pyenv, etc.
```

Comment:
Do you want to mention the `conda` way? Also with pyenv, it installs specific versions of Python to use and doesn't manage virtual environments

Reply:
Nevermind, there is a whole conda doc... maybe link to it from here?